modules/ROOT/pages/logs-using.adoc

```diff
@@ -208,8 +208,12 @@ The default settings are:
 
 == Working with Predefined Filters
 
-When the Quick Filter list is open, you can select and use predefined log
-filters. This example selects the ERROR log level.
+When the *Quick Filters* list is open, you can select and use predefined log filters.
+
+[NOTE]
+When there are many predefined filters available, you can select *See More* to further filter your results. The *See More* list displays the first five results of the most recent 500 log entries.
+
```

Comment:
The NOTE is not 100% accurate. I would rather write:
NOTE:
Quick Filters result list displays-for every result group-the first five results of the most recent 500 log entries.
When there are more than 500 hits or the top 5 values does not represent 100% of total values, a "See More" action for each result group appears. Clicking the "See more" link will open the "Add a Filter" box, and pre-set it to the appropriate parameters.